docs/authentication/social-connections/custom-provider.mdx

```diff
@@ -0,0 +1,78 @@
+---
+description: Configure a custom OpenID Connect (OIDC) compatible authentication provider for your Clerk application.
+---
+
+# OAuth with a custom auth provider
+
+<Callout type="warning">
+  Custom auth providers are in beta. Please [contact support](/contact/support) to get your application whitelisted for
+  this feature.
+</Callout>
+
+Clerk allows you to configure custom OpenID Connect (OIDC) compatible authentication providers for your application. This guide walks you through the steps to set up a custom OAuth provider.
```

Comment:
Let's move the tutorial hero to the top.

```suggestion
<TutorialHero
  beforeYouStart={[
    {
      title: "A Clerk application is required.",
      link: "/docs/quickstarts/setup-clerk",
      icon: "clerk",
    },
    {
      title: "An OIDC identity provider is required.",
      link: "https://openid.net/specs/openid-connect-core-1_0.html",
      icon: "user-circle",
    }
  ]}
>
  - Configure custom Auth provider for your Clerk application.
</TutorialHero>

Clerk allows you to configure custom OpenID Connect (OIDC) compatible authentication providers for your application. This guide walks you through the steps to set up a custom OAuth provider.
```

Reply:
If I get this right, you did this change here: [`c107eb9` (#1321)](https://github.com/clerk/clerk-docs/pull/1321/commits/c107eb9d3322a22576e0715ad6cd43adeb0f8fb0)

So I'm resolving this.